README.rst

```diff
@@ -0,0 +1,162 @@
+      settings.
+
+   b) Repeat steps 1-7 above, but in Step 2, use ``util/setchplenv.*``
+      instead of ``util/quickstart/setchplenv.*``
+
+   This will set up your environment to use Chapel in the preferred
+   configuration.  Making this configuration will involve building one
+   or more third-party packages which will increase the overall build
+   time.  If you run into any portability issues, please let us know
+   at chapel_info@cray.com.
+
+
+9) If you plan to do performance studies of Chapel programs, be sure
+   to (a) use the full-featured version and (b) read the ``PERFORMANCE``
+   file in this directory to avoid common pitfalls.
+
+What's next?
+------------
+=============================================== =====================================
+**For more detailed information about:**        **refer to:**
+----------------------------------------------- -------------------------------------
+    changes since the last release              ``CHANGES``
+    implementation status                       ``STATUS``
+    performance status                          ``PERFORMANCE``
+    Chapel's file and directory structure       ``README.files``
+    prerequisites for using this release        ``doc/prereqs.rst``
+    setting Chapel environment variables        ``doc/chplenv.rst``
+    building the compiler                       ``doc/building.rst``
+    using the compiler                          ``doc/compiling.rst``
+    executing Chapel programs                   ``doc/executing.rst``
+    debugging Chapel programs                   ``doc/debugging.rst``                  
+    reporting bugs                              ``doc/bugs.rst``
+    Chapel modes for emacs and vim              ``etc/README.md``
+    example Chapel programs                     ``examples/README``
+    a quick reference for Chapel syntax         ``doc/quickReference.pdf``
+    the Chapel language specification           ``doc/chapelLanguageSpec.pdf``
+    executing programs on multiple locales      ``doc/multilocale.rst``
+    how Chapel tasks are mapped to threads      ``doc/tasks.rst``
+    calling C code from Chapel                  ``doc/technotes/extern.rst``
+    other user and technical notes              ``doc/README``
+    third-party software that we use            ``third-party/README``
+=============================================== =====================================
+
+
+=============================================== =====================================
+**To use Chapel on a specific platform:**       **refer to:**
+----------------------------------------------- -------------------------------------
+       a Cray system                            ``doc/platforms/cray.rst``
+       Cygwin over Windows                      ``doc/platforms/cygwin.rst``
+       an IBM system                            ``doc/platforms/ibm.rst``
+       an SGI Altix system                      ``doc/platforms/sgi.rst``
```

Comment:
Let's drop these final two entries given that we have low confidence in them and are not deeply invested in them.  As we improve download.html, we could potentially remove the first two as well, but they're important/unique enough that I think we should keep them for now.